docs/src/publications.md

```diff
@@ -6,6 +6,26 @@ Pages = ["publications.md"]
 
 This page lists publications about the JuliaReach ecosystem and its applications.
 
+## ARCH 2019 Competition NLN Category Report
+
+- **ARCH-COMP19 Category Report: Continuous and Hybrid Systems with Nonlinear Dynamics.** Fabian Immler, Matthias Althoff, Luis Benet, Alexandre Chapoutot, Xin Chen, Marcelo Forets, Luca Geretti, Niklas Kochdumper, David P. Sanders and Christian Schilling (2019) ARCH19. 6th International Workshop on Applied Verification of Continuous and Hybrid Systems, vol 61, pages 41--61 [doi: 10.29007/bj1w](https://easychair.org/publications/paper/1gbP). Packages: [Reachability.jl](https://github.com/JuliaReach/Reachability.jl).
```

Comment:
Should we mention the cooperation with the `Taylor*` packages?

Reply:
This is a good idea. Actually we can add the paragraph describing JuliaReach here.

Reply:
Good, we can add explicit links as well.